docs/rules/README.md

```diff
@@ -86,6 +88,8 @@ Enforce all the rules in this category, as well as all higher priority rules, wi
 | [vue/return-in-computed-property](./return-in-computed-property.md) | enforce that a return statement is present in computed property |  |
 | [vue/return-in-emits-validator](./return-in-emits-validator.md) | enforce that a return statement is present in emits validator |  |
 | [vue/use-v-on-exact](./use-v-on-exact.md) | enforce usage of `exact` modifier on `v-on` |  |
+| [vue/valid-define-emits](./valid-define-emits.md) | enforce valid `defineEmits` compiler macro |  |
+| [vue/valid-define-props](./valid-define-props.md) | enforce valid `defineProps` compiler macro |  |
 | [vue/valid-template-root](./valid-template-root.md) | enforce valid template root |  |
```

Comment:
You might want to include `vue/valid-next-tick` here, too: https://github.com/vuejs/eslint-plugin-vue/blob/v7.17.0/lib/rules/valid-next-tick.js#L122

Reply:
I added it. Thank you!